brave/browser-laptop
# Fix untranslated recovery code text

## Submitter Checklist:
Closes #13311 


## Test Plan:
1. Set browser language to any other apart from `English US`
2. Enable payments -> Advance settings-> Backup codes
3. Print recovery codes should open key in a new tab and should read `Recovery Code`
4. Save recovery key and open in editor, should read `Recovery Code`

## Reviewer Checklist:


Tests
